edly-io/edx-platform
# Hotjar and GA Integration

**Description:** 
Add Hotjar and GA tracking codes in LMS to enable learner user tracking across site.

**JIRA:** 
https://edlyio.atlassian.net/browse/EDLY-3972

**Testing instructions:**

1. Check that the Hotjar is showing user data for green dev
2. Check that user data is shown on Google Analytics Green Dev Property

**Merge checklist:**


**Post merge:**


**Note:** Please add screenshots for any view able change.
